core/src/main/scripts/importer/validateData.py

```diff
@@ -1622,58 +1607,44 @@ class ClinicalValidator(Validator):
             'datatype': 'STRING'
         },
         'PRIMARY_SITE': {
-            'is_patient_attribute': '0',
             'datatype': 'STRING',
-            'when_wrong': 'WARNING'
         },
         'SAMPLE_CLASS': {
-            'is_patient_attribute': '0',
             'datatype': 'STRING'
         },
         'SAMPLE_DISPLAY_NAME': {
             'is_patient_attribute': '0',
             'datatype': 'STRING'
         },
         'SAMPLE_TYPE': {
-            'is_patient_attribute': '0',
```

Comment:
by removing this, does it mean is can now also be a patient attribute? If that is the case, then I don't think this makes sense. All `SAMPLE_` attributes should remain sample type attributes IMO.

Reply:
`'is_patient_attribute': '0',` will make it a SAMPLE attribute. Removing `'is_patient_attribute'`will make it flexible.

I've updated the commit, now added all SAMPLE_*-related attributes to the core sample set.